ext/kv/01_db.ts

```diff
@@ -764,4 +751,30 @@ class KvListIterator extends AsyncIterator
   }
 }
 
+async function doAtomicWriteInPlace(
+  rid: number,
+  checks: [Deno.KvKey, string | null][],
+  mutations: [Deno.KvKey, string, RawValue | null, number | undefined][],
+  enqueues: [Uint8Array, number, Deno.KvKey[], number[] | null][],
+): Promise<string | null> {
+  for (const m of mutations) {
+    const key = m[0];
+    if (
+      key.length && m[1] === "set" &&
+      key[key.length - 1] === commitVersionstampSymbol
```

Comment:
what if `commitVersionstamp()` is used as not the last part of the key? or if its used with operation other than "set"? seems like we should throw in that case?

Reply:
Native code checks for that as part of deserialization, and throws

Reply:
Added to the test